django_gcloud_storage/__init__.py

```diff
@@ -128,6 +128,11 @@ def __init__(self, project=None, bucket=None, credentials_file_path=None):
         else:
             self.project_name = settings.GCS_PROJECT
 
+        if unsigned_urls is not None:
+            self.unsigned_urls = unsigned_urls
```

Comment:
This can simply be set to True since the actual value of unsigned_urls in the constructors parameter is irrelevant.